packages/fuel-indexer-schema/src/lib.rs

```diff
@@ -260,9 +262,9 @@ impl FtColumn {
                 None => String::from(NULL_VALUE),
             },
             FtColumn::Blob(value) => match value {
-                Some(val) => {
+                Some(fuel_indexer_types::Blob(val)) => {
                     let x = hex::encode(val);
-                    format!("'{x}'")
+                    format!("'{}'", x)
```

Comment:
I think the previous way is preferred https://rust-lang.github.io/rust-clippy/master/